src/style/mixins.scss

```
@@ -167,8 +167,12 @@
   }
 }
 
-@mixin iui-scroll-snapping {
+@mixin iui-scroll-snapping($selector: '> *') {
```

Can you add some comments here? It's not obvious what exactly this mixin does without looking at the code

You can use the mixins in `labeled-inputs.scss` as example of how to format

---

Added `/// Mixin allows scroll snapping to individual table rows if .iui-scroll-snapping is applied to the table`

---

uhhh that's not what I meant 😆 The comment needs to document the mixin itself (in `mixins.scss`), not its reference

The tripe slash comment format serves as ["SassDoc"](http://sassdoc.com/) (we don't do anything with it currently, but it's nice).

https://github.com/iTwin/iTwinUI/blob/2bb38fb392dd459707e02500436f2f98166a573b/src/inputs/labeled-inputs.scss#L275-L279

---

You can just commit this if it's easier 😄 

```suggestion
/// Adds the ability to toggle vertical scroll snapping using .iui-scroll-snapping.
/// @arg $selector - selector to apply `scroll-snap-align: start` on. Defaults to '> *'
@mixin iui-scroll-snapping($selector: '> *') {
```

---

Ah, yes I did misunderstand.  I like your suggestion, added.